### DISCUSSION TITLE
Show homepage image in Twitter card when sharing a post

### DISCUSSION THREAD

**Discussion by szilardszaloki** [mmistakes/jekyll-theme-basically-basic#116] [Q&A] [answered]
Hi there,
There's an image on my homepage which I'd like to be shown on my Twitter card when I share a post. At the same time, I don't want this image to be shown in any of my posts, that is I don't want to include this:
```
image: /assets/images/whatever.png
```
in any of my posts' front matter.
Is there a way to achieve this?

Thanks in advance!

**Comment by mmistakes** [comment, marked as answer]
The theme uses the Jekyll-seo-tag plugin for adding Twitter card meta. Check their docs if it’s possible to assign an image path for just that.

https://github.com/jekyll/jekyll-seo-tag/blob/master/docs/README.md

**Comment by szilardszaloki** [comment]
Hi Michael,
Thanks for pointing me in the right direction!
